qutip/solver/__init__.py

```diff
@@ -1,11 +1,17 @@
 from .result import *
 from .options import *
-from .integrator import *
-from .ode import *
+import qutip.solver.integrator
+from .integrator import IntegratorException
```

Comment:
What are the imports of `qutip.solver.integrator` and `qutip.solver.ode` for? I assume they are to make the integrators accessible without directly exposing them in the `qutip.solver` namespace? If that is the case, I think we should make that more explicit and ensure that they're not exposed here at all (e.g. import them as `_integrator` and maybe `del _integrator` immediately after).

Reply:
I want them available in a less direct way, I moved `integrator.py` to the `ode` folder and renamed it `integrator`.
Thus they are available as `qutip.integrator.Integrator...` instead of `qutip.Integrator...`.

Reply:
Sounds good.